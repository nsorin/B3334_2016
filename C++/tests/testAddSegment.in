LIST
S s1 12 15 45 20
LIST
EXIT
