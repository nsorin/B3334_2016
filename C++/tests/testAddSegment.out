OK
S s1 12 15 45 20
